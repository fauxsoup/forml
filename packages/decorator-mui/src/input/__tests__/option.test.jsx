import Option from '../option';
import React from 'react';
import { render } from '@testing-library/react';

describe('renders', function () {
    let form;

    beforeEach(function () {
        let form = { type: 'number', enum: [1, 2, 3] };
    });

    test('consistently', function () {
        const { container } = render(<Option value={1}>1</Option>);

        expect(container).toMatchSnapshot();
    });
});
