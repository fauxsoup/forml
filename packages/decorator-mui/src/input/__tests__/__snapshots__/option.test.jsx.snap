// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders consistently 1`] = `
<div>
  <li
    class="MuiButtonBase-root MuiMenuItem-root MuiMenuItem-gutters MuiMenuItem-root MuiMenuItem-gutters css-kk1bwy-MuiButtonBase-root-MuiMenuItem-root"
    role="menuitem"
    tabindex="-1"
    value="1"
  >
    1
    <span
      class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
    />
  </li>
</div>
`;
